test/e2e/models/CodeServer.ts

```diff
@@ -136,7 +134,7 @@ export class CodeServer {
       })
 
       proc.on("close", (code) => {
-        const error = new Error("closed unexpectedly")
+        const error = new Error("code-server closed unexpectedly")
```

Comment:
🔥

Reply:
lol yeah I saw "closed unexpectedly" when I was debugging and had no idea what closed